From c92b009ef91be7b5f09d8174de08c2de0b6bb4ec Mon Sep 17 00:00:00 2001
From: make shi <b15407@freescale.com>
Date: Thu, 23 Aug 2012 17:07:13 +0800
Subject: [PATCH 2381/2463] ENGR00220096 USB core: remove Logitech Quickcam
 E3500 form usb_quirk_list

Since Logitech Quickcam E3500 is defaultly listed in usb_quirk_list on current
linux kernel. So the USB camera only supports reset resume, but doesn't support
normal usb suspend/resume. Actually, the USB camera works abnormally after USB
reset resume, but it works well after doing normal suspend/resume.

Signed-off-by: make shi <b15407@freescale.com>
---
 drivers/usb/core/quirks.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/drivers/usb/core/quirks.c b/drivers/usb/core/quirks.c
index 32d3adc..1b5eaa0 100644
--- a/drivers/usb/core/quirks.c
+++ b/drivers/usb/core/quirks.c
@@ -80,9 +80,6 @@ static const struct usb_device_id usb_quirk_list[] = {
 	/* Logitech Quickcam Pro 9000 */
 	{ USB_DEVICE(0x046d, 0x0990), .driver_info = USB_QUIRK_RESET_RESUME },
 
-	/* Logitech Quickcam E3500 */
-	{ USB_DEVICE(0x046d, 0x09a4), .driver_info = USB_QUIRK_RESET_RESUME },
-
 	/* Logitech Quickcam Vision Pro */
 	{ USB_DEVICE(0x046d, 0x09a6), .driver_info = USB_QUIRK_RESET_RESUME },
 
-- 
1.7.7.4

